examples/checkbox/checkbox-1/checkbox-1.html

```diff
@@ -37,8 +37,8 @@ <h2 id="ex_label">Example</h2>
       <div role="separator" id="ex_start_sep" aria-labelledby="ex_start_sep ex_label" aria-label="Start of"></div>
 
       <div  id="ex1">
-        <h3>Sandwich Condiments</h3>
-        <div>
+        <h3 id="id-group-label">Sandwich Condiments</h3>
+        <div role="group" aria-labelledby="id-group-label">
```

Comment:
Sorry I missed this before ... the states and properties table says that the group role is on the UL element. It is actually on a div here. I'm not sure which is better. The advantage of the div is the list semantics that says there are 4 checkboxes is preserved. The disadvantage is that is just more verbage on the page that may not be particularly valuable. I could go either way. We just need the documentation and the code to be consistent.